### PR TITLE
LPS-134558, LPS-143145 Auto SF

### PR DESCRIPTION
**Pull request by ling-alan-huang**

### CODE REVIEW

File: modules/apps/frontend-theme/frontend-theme-dialect-style-guide-sample-web/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"dependencies": {
-		"@clayui/card": "3.40.0",
+		"@clayui/card": "3.41.0",
```
Comment (ling-alan-huang) [inline modules/apps/frontend-theme/frontend-theme-dialect-style-guide-sample-web/package.json:3]: @ethib137 Can SF auto change this?

**Comment by ling-alan-huang** [comment]
ci:forward

File: modules/apps/frontend-theme/frontend-theme-dialect-style-guide-sample-web/package.json (hunk 1 above)
Comment (ling-alan-huang) [reply modules/apps/frontend-theme/frontend-theme-dialect-style-guide-sample-web/package.json:3]: Here is your changes: https://github.com/liferay/liferay-portal/commit/08b5fc1115a91e8a5960a0d432d6658522100b34

Comment (ethib137) [reply modules/apps/frontend-theme/frontend-theme-dialect-style-guide-sample-web/package.json:3]: Hey @ling-alan-huang, whether it's safe to automatically update Clay dependencies is something that should be checked with the frontend infrastructure team. In this specific case, it's probably not going to break anything, but I don't know if that will always be the case.

**Pull request closed without merging**